### DISCUSSION TITLE
Vite dev server behind a reverse proxy

### DISCUSSION THREAD

**Discussion by gsguglielmo** [laravel/framework#43262] [Ideas]
I've an issue when running a Vite dev server behind a reverse proxy. 
If the dev server is running (the `public/hot` file is present), the `@vite` directive, will automatically get the base url for the tag from the `hot` file, which is fine, if you don't have a reverse proxy in front of the dev server. In my case the `hot` file will always have an IPv4/IPv6 address, even if you change the settings of the vite.config.js with a hostname, it will resolve the hostname. 
![image](https://user-images.githubusercontent.com/8240218/179601928-832a6143-ef77-4f56-8cba-b9e6afa10dc6.png)

It would be nice to have a config to manually set the base url, just like Mix. Mix has a config `app.mix_hot_proxy_url` which will use the specified url if present and continue with the default behavior if not.
https://github.com/laravel/framework/blob/b15dc2ac7e4c7b382ec5cf09e2983ab94cccc83c/src/Illuminate/Foundation/Mix.php#L35-L39

I'm willing to make a pull request if you think that this would be a good feature to have 🙂

**Comment by julianlaibach** [comment]
Hey @gsguglielmo do you have any workaround for this? I cannot figure out how to get this working...
Thanks a lot!
Best,
Julian
